certbot/tests/cli_test.py

```diff
@@ -186,10 +186,6 @@ def test_help(self):
         self.assertTrue("--delete-after-revoke" in out)
         self.assertTrue("--no-delete-after-revoke" in out)
 
-        out = self._help_output(['-h', 'config_changes'])
```

Comment:
nit: The assertions that `--cert-path` and `--key-path` aren't included in the help output are mildly useful. Can we keep this code but change the subcommand used here from `config_changes` to something like `register`?